desktop/core/src/desktop/js/reactComponents/AlertComponent/AlertComponent.scss

```diff
@@ -27,14 +27,17 @@
   flex-flow: column;
   row-gap: 10px;
 
+  .ant-alert {
```

Comment:
as we are overriding the ant-alert class it might affect other places where we use or might use ant-alert., would be good to wrap this around a custom alert class specific to hue.

Reply:
Yes that is a good practise, and if we can add a custom class to the ant-alert component it would be cleaner that overriding it, **but this overriding** is already wrapped by the .hue-alert.